doc/manifest/schema/1.1.0/defaultLocale.md

```diff
@@ -0,0 +1,292 @@
+  
+  If this is a link to the copyright file for an open source project, it should be specific to the version for the package. Some open source projects change their copyright over time.
+ </details>
+
+### ShortDescription
+
+<details>
+  <summary>The short package description</summary>
+
+  **Required Field**
+  This key represents the description for a package. It is intended for use in `winget show` to help a user understand what the package is.
+
+  >Note: This should be something descriptive about what the package does, and it should not simply state something like "&lt;package name&gt; installer" or "&lt;package name&gt; setup".
+ </details>
+ 
+### Description
+
+<details>
+  <summary>The full package description</summary>
+
+  **Optional Field**
+  This key represents the full or long description for a package. It is *not* currently used in the Windows Package Manager.
+
+  >Note: This was included for future integration with the Microsoft Store source to provide the ability to display the full package description.
+ </details>
+ 
+### Moniker
+
+<details>
+  <summary>The most common package term</summary>
+
+  **Optional Field**
+  This key represents the most common term users would search for when installing or upgrading a package. If only one package uses this moniker, then the [install](https://docs.microsoft.com/windows/package-manager/winget/install), [list](https://docs.microsoft.com/windows/package-manager/winget/list) and [upgrade](https://docs.microsoft.com/windows/package-manager/winget/upgrade) command may match with this package. 
+  
+  >Note:Moniker is the third property evaluated when searching for a matching package.
+ </details>
+ 
+### Tags
+
+<details>
+  <summary>List of additional package search terms</summary>
+
+  **Optional Field**
+  This key represents other common term users would search for when looking for packages. 
+
+  >Note: The best practice is to present these terms in all lower case with hyphens rather than spaces.
+ </details>
+
+ ### Agreement
+ 
+ <details>
+   <summary>List of package agreements</summary>
+
+   **Optional Field**
+   This key holds any agreements a user must accept prior to download and subsequent install or upgrade.
+
+   >Note: In the Windows Package Manager Community Repository, these are only allowed to be submitted by verified developers.
+  </details>
+
+#### AgreementLabel
+  
+<details>
+  <summary>The label for a package agreement</summary>
+    
+  **Optional Field**
+  This key represents the label for a package agreement.
+</details>
+  
+#### Agreement
+
+<details>
+  <summary>The text for a package agreement</summary>
+    
+  **Optional Field**
+  This key represents the text or body of a package agreement.
+</details>
+
+#### AgreementUrl
+
+<details>
+  <summary>The URL for a package agreement</summary>
+  
+  **Optional Field**
+  This key represents the URL for a package agreement.
+</details>
```

Comment:
I believe you can nest details inside of details

Reply:
Yeah, I didn't catch the parent and the child having the same name. We might consider changing the parent to "Agreements" for disambiguation in a future breaking schema change for 2.0.